app/src/main/res/layout/fragment_user_details.xml

```diff
@@ -0,0 +1,110 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:padding="@dimen/default_separator_margin"
+    tools:context="com.globant.example.mentorapp.subscriberDetails.presentation.view.fragment.UserDetailsFragment">
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal">
+
+        <ImageView
+            android:id="@+id/imageView"
+            android:layout_width="@dimen/detail_user_image_size"
+            android:layout_height="@dimen/detail_user_image_size"
+            app:srcCompat="@drawable/ic_placeholder"
+            tools:ignore="ContentDescription" />
+
+        <TextView
+            android:id="@+id/txtUserName"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignParentTop="true"
+            android:layout_marginStart="@dimen/list_general_margin"
+            android:layout_toEndOf="@+id/imageView"
+            tools:text="@string/tv_detail_user_name" />
+
+        <TextView
+            android:id="@+id/txtLocation"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/txtUserName"
+            android:layout_marginStart="@dimen/list_general_margin"
+            android:layout_toEndOf="@+id/imageView"
+            tools:text="@string/tv_detail_location" />
+
+        <TextView
+            android:id="@+id/txtCompany"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/txtLocation"
+            android:layout_marginStart="@dimen/list_general_margin"
+            android:layout_toEndOf="@+id/imageView"
+            android:text="@string/tv_detail_company"
+            tools:text="@string/tv_detail_company" />
+
+
+    </RelativeLayout>
+
+    <View
+        style="@style/divider"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/divider_height"
```

Comment:
You can set this property in your _style/divider_ and also remove width and height from this element.
Also, margin properties can be moved to that style as well.